miqa/urls.py

```diff
@@ -36,8 +39,9 @@
     permission_classes=(permissions.IsAuthenticated,),
 )
 
+
 urlpatterns = [
-    path('', HomePageView.as_view(), name='home'),
+    settings.HOMEPAGE,
```

Comment:
```suggestion
    path('', settings.HOMEPAGE_VIEW, name='home')
```
I think that's a little more DRY, and makes it a little clear that you're shoving a view into a setting. It also simplifies the properties a bit.

Reply:
I'm getting a strange error when I separate them like that:
`TypeError: Couldn't setup configuration 'miqa.settings.DevelopmentConfiguration':  view() missing 1 required positional argument: 'request'`.

I get this when urlpatterns contains `path('', sesttings.HOMEPAGE_VIEW, name='home)` and settings.HOMEPAGE_VIEW returns `RedirectView.as_view(url='https://localhost:8081')`. It's as if having them separate breaks how `as_view` works with `path` to receive a request?